meeting-notes/2018/2018-09-24--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,176 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 September 24, 2018
+
+- **Lead:** Jacob Heun
+- **Notetaker:** @alanshaw
+- **Attendees:**
+  - _attendee names..._
+  - @alanshaw
+  - @achingbrain
+  - @jacobheun
+  - @hugomrdias
+  - @momack2
+  - @vasco-santos
+  - @vmx
+  - @victorbjelkholm
+  - @pgte
+  - @mikeal
+- **Recording:** https://youtu.be/9_p7GKP_e5A
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+- **DEMO**
+	- Alan Shaw - spawning IPFS daemons concurrently with a port service
+  - Victor - Less of a demo: new js pipeline https://github.com/ipfs/jenkins-libs/pull/35
```

Comment:
Wait, there were demos? I don't see them on the recording and also, these should go on the IPFS All Hands as this call is designed for high throughput sync update

@alanshaw @VictorBjelkholm

Reply:
We ran out of time so they did not happen!

Reply:
Let's save demos for the IPFS All Hands :)